src/prefect/deployments/runner.py

```diff
@@ -476,3 +612,188 @@ async def from_storage(
         cls._set_defaults_from_flow(deployment, flow)
 
         return deployment
+
+
+class DeploymentImage:
+    def __init__(self, name, tag=None, dockerfile="auto", build_kwargs=None):
```

Comment:
minor stylistic nit
```suggestion
    def __init__(self, name, tag=None, dockerfile="auto", **build_kwargs):
```

Reply:
I was trying to be clever and leave room for `push_kwargs`, but I realize now there isn't anything users should need to override when pushing images.